sensors/ev3_uart_sensor_defs.c

```diff
@@ -109,7 +109,7 @@ const struct ev3_uart_sensor_info ev3_uart_sensor_defs[] = {
 				 * the EV3 brick. As a result, the sensor will time out and reset.
 				 *
 				 * @name_footnote: [^cal-mode]
-				 * @description: Calibration ???<br />All LEDs rapidly cycling, appears white
+				 * @description: Calibration ???<br />Red LED flashing every 4 seconds, then goes continous
```

Comment:
Interesting. I don't think I ever really tried this mode since the sensor resets itself after a few seconds.